api/aggregators/searchAggregator.js

```diff
@@ -108,10 +108,25 @@ exports.createMatchAggr = async (schemaName, projectId, keywords, caseSensitive,
  */
 exports.createSortingPagingAggr = function(schemaName, sortValues, sortField, sortDirection, pageNum, pageSize) {
   const searchResultAggregation = [];
-  let defaultTwoSorts = false;
+  let datePostedHandlingTruncating = false;
+  if (sortField && sortField.includes(",") || Object.keys(sortValues).length > 1){
+    //sort will have multiple values passed
+    if (sortField.includes("datePosted") || sortValues.hasOwnProperty("datePosted")){
```

Comment:
Calling `hasOwnProperty` in this way will fail linting. You have it correct further down.

Reply:
Changed.